addons/auth/package.json

```diff
@@ -15,7 +15,7 @@
     "format": "yarn run format:js",
     "format:js": "prettier --write '{addon,app,config,stories,tests}/**/*.js' *.js",
     "start": "ember serve",
-    "test": "ember test",
+    "test": "ember test --test-port 0",
```

Comment:
NOT blocking comment: As I saw in other package.json, defining `--test-port` to 0 means the framework will choose a port for each suite? just wondering, because if we assign same port to let's say auth and core, if they run at the same time it will fail?

Reply:
Yes, it's so we don't run them all on the same port at the same time which will conflict.